### PR TITLE
Updating Vungle to 6.10.3

### PR DESCRIPTION
**Pull request by jiancchen**
#### Version 6.10.3.0
- Verified compatibility with Vungle SDK 6.10.3.
- Added support for Child-Directed setting (COPPA) to be pass to the Vungle SDK via new API

Built and tested with:
- Google Mobile Ads SDK version 20.5.0.
- Vungle SDK version 6.10.3.

### CODE REVIEW

File: ThirdPartyAdapters/vungle/CHANGELOG.md [hunk 1]
```diff
@@ -1,152 +1,238 @@
...
-- Verified compatibility with Vungle SDK 5.3.0.
-
-#### Version 5.1.0.0
-- Updated the adapter to make it compatible with Vungle SDK 5.1.0.
-- Changed the version naming system to
-  [Vungle SDK version].[adapter patch version].
-
-#### Earlier versions
-- Added support for interstitial and rewarded video ad formats.
+## Changelog
```
Comment (joshuajplagonera) [inline ThirdPartyAdapters/vungle/CHANGELOG.md:1]: Please revert this file. I believe these are Vungle SDK specific changelog notes and not necessarily for the adapter.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java [hunk 2]
```diff
@@ -109,6 +114,17 @@ public void onAutoCacheAdAvailable(String placementId) {
...
 
+  public void updateCoppaStatus(int configuration) {
+    switch (configuration) {
+      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE :
+        Vungle.updateUserCoppaStatus(true);
+        break;
+      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE :
+        Vungle.updateUserCoppaStatus(false);
+        break;
+    }
```
Comment (joshuajplagonera) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:125]: Please add a `default` case for readability.

**Review by joshuajplagonera — CHANGES_REQUESTED** [review]

File: ThirdPartyAdapters/vungle/CHANGELOG.md (hunk 1 above)
Comment (jiancchen) [reply ThirdPartyAdapters/vungle/CHANGELOG.md:1]: @joshuajplagonera We are trying to have a consistent release notes w/ each version. Yes, there are Vungle specific behavior changes in the logs but they apply to the adapters as well. Is there any part you guys want removed from the changelog and if there are, do you have suggestions as to what you to replace it?

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java (hunk 2 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:125]: Gentle bump on this.

File: ThirdPartyAdapters/vungle/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 ## Vungle Android Mediation Adapter Changelog
 
+#### Version 6.10.3.0
+- Verified compatibility with Vungle SDK 6.10.3.
+- Added support for Google’s [Android 12](https://developer.android.com/about/versions/12).
+- SDK now supports Android [Target API 31](https://developer.android.com/sdk/api_diff/31/changes).
```
Comment (joshh-devrel) [inline ThirdPartyAdapters/vungle/CHANGELOG.md:6]: +1 on Joshua's previous comment to remove those that are non-adapter changes.

This changelog should only contain changes in the adapter but not the SDK. Publishers should not refer this changelog for changes in the SDK.

Comment (jiancchen) [reply ThirdPartyAdapters/vungle/CHANGELOG.md:6]: @joshh-devrel
What would look acceptable in this case?
i.e.?
#### Version 6.10.3.0
- Verified compatibility with Vungle SDK 6.10.3.

Built and tested with:
- Google Mobile Ads SDK version 20.5.0.
- Vungle SDK version 6.10.3.

Comment (joshuajplagonera) [reply ThirdPartyAdapters/vungle/CHANGELOG.md:6]: This looks fine to me.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java (hunk 2 above)
Comment (jiancchen) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:125]: @joshuajplagonera @joshh-devrel 

Gotcha, quick question on the behavior for COPPA:

Couldn't find guidelines, but from what I read it looks like we should:

TAG_FOR_CHILD_DIRECTED_TREATMENT_UNSPECIFIED: treat as COPPA false?
default: treat as COPPA true?

Does that look correct?

Comment (preethi-sannathi) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:125]: @joshuajplagonera  Please specify how Google treats TAG_FOR_CHILD_DIRECTED_TREATMENT_UNSPECIFIED users. Are these users treated as Non-COPPA?

Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:125]: @jiancchen @preethi-sannathi 

As illustrated in [this dev doc](https://developers.google.com/android/reference/com/google/android/gms/ads/mediation/MediationAdConfiguration#constants):
- `TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE`: indicates that the app should not be treated as child-directed for purposes of the Children’s Online Privacy Protection Act (COPPA).
- `TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE`: indicates that the app should be treated as child-directed for purposes of the Children’s Online Privacy Protection Act (COPPA).
- `TAG_FOR_CHILD_DIRECTED_TREATMENT_UNSPECIFIED`: indicates that the publisher has not specified whether the app should be treated as child-directed for purposes of the Children’s Online Privacy Protection Act (COPPA).

Comment (jiancchen) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:129]: @joshh-devrel 
@joshuajplagonera updated behavior thanks

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java [hunk 4]
```diff
@@ -109,6 +114,21 @@ public void onAutoCacheAdAvailable(String placementId) {
     // Unused
   }
 
+  public void updateCoppaStatus(int configuration) {
+    switch (configuration) {
+      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE :
+        Vungle.updateUserCoppaStatus(true);
+        break;
+      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE :
```
Comment (joshh-devrel) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:122]: Question: Do we need to do `Vungle.updateUserCoppaStatus(false);` in this case well?

File: ThirdPartyAdapters/vungle/CHANGELOG.md [hunk 5]
```diff
@@ -135,10 +142,10 @@ Built and tested with:
...
 - Updated the Adpater project for Android Studio 3.0.
 - Added the following methods to Bundle builder class.
-   - `setOrdinalViewCount` : This field is used to pass the mediation ordinal,
-   whenever Publisher receives the ordinal data reports from Vungle.
-   - `setFlexViewCloseTimeInSec` : This option is used to make flex view ads
-   dismiss on their own after the specified number of seconds.
+    - `setOrdinalViewCount` : This field is used to pass the mediation ordinal,
+      whenever Publisher receives the ordinal data reports from Vungle.
+    - `setFlexViewCloseTimeInSec` : This option is used to make flex view ads
+      dismiss on their own after the specified number of seconds.
```
Comment (joshh-devrel) [inline ThirdPartyAdapters/vungle/CHANGELOG.md:148]: Nit: Please fix the indentation.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java (hunk 4 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java:122]: Never-mind, it's a fall-through.

File: ThirdPartyAdapters/vungle/CHANGELOG.md (hunk 5 above)
Comment (joshh-devrel) [reply ThirdPartyAdapters/vungle/CHANGELOG.md:148]: We can fix it internally.

**Pull request merged**